modules/console/pages/config/security/authentication.adoc

```diff
@@ -19,25 +19,31 @@
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-This topic describes how to enable authentication in Redpanda Console and explains how it interacts with the Kafka API, Admin API, and Schema Registry API in Redpanda.
+This topic describes how to enable authentication in Redpanda Console and how it integrates with the Kafka, Admin, and Schema Registry APIs in Redpanda.
 
-Authentication in Redpanda Console lets users log in and optionally forward credentials to the connected cluster, ensuring API requests run under the user's identity. Redpanda Console supports two authentication methods:
+Authentication in Redpanda Console lets users log in and optionally forward their credentials to the connected cluster. This ensures API requests run under the user's identity.
 
-* **OIDC (OpenID Connect):**
-  Uses external identity providers, such as Google, to facilitate single sign-on (SSO).
-* **Basic authentication:**
-  Provides a traditional username and password login.
+Redpanda Console supports the following authentication methods:
+
+* **OIDC (OpenID Connect):** Integrates with external identity providers (IdPs) for single sign-on (SSO).
+* **Basic authentication:** Uses traditional username and password credentials.
 
 image::console:login.png[]
 
-== How authentication works in Redpanda Console
+== How authentication works
```

Comment:
This is not generally about authN--it's about how authN works _in RP Console_. I imagine the more generic heading may come up in Algolia or AI searches for general authN queries. I would keep the original here.